components/eventing-controller/pkg/handlers/jetstream_test.go

```diff
@@ -1165,9 +1165,8 @@ func getJetStreamClient(t *testing.T, serverURL string) *jetStreamClient {
 	}
 }
 
-func addJSCleanEventTypesToStatus(sub *eventingv1alpha1.Subscription, cleaner eventtype.Cleaner, jsBackend *JetStream) {
-	cleanedSubjects, _ := GetCleanSubjects(sub, cleaner)
-	sub.Status.CleanEventTypes = jsBackend.GetJetStreamSubjects(cleanedSubjects)
+func addJSCleanEventTypesToStatus(sub *eventingv1alpha1.Subscription, cleaner eventtype.Cleaner) {
+	sub.Status.CleanEventTypes, _ = GetCleanSubjects(sub, cleaner)
```

Comment:
I see it was like that before your changes, but I don't think we should ignore errors here.

Reply:
I have updated it.